src/libs/OptionsListUtils.ts

```diff
@@ -1617,11 +1623,19 @@ function createOptionFromReport(report: Report, personalDetails: OnyxEntry<Perso
  * @param searchValue - search string
  * @returns a sorted list of options
  */
-function orderOptions(options: ReportUtils.OptionData[], searchValue: string | undefined, {preferChatroomsOverThreads = false} = {}) {
+function orderOptions(options: ReportUtils.OptionData[], searchValue: string | undefined, {preferChatroomsOverThreads = false, preferPolicyExpenseChat = false} = {}) {
     return lodashOrderBy(
         options,
         [
             (option) => {
+                if (option.isPolicyExpenseChat && preferPolicyExpenseChat && option.policyID === activePolicyID) {
+                    return 0;
+                }
+
+                if (option.isPolicyExpenseChat && preferPolicyExpenseChat) {
```

Comment:
@mountiny I have a question here: Do we also prefer re-order the other policy expense chat that isn't the active policy ID when we re-order the active policy expense chat.

Reply:
Actually, I implemented this logic. If we don't want to do that, I can remove this logic.